<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>93</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>456</width>
    <height>93</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>ONe</string>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
      <item>
       <property name="text">
        <string>ONe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arun</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Anu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Anubhav</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Three</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>hey</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>hi</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::YearSection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
